src/services/manage/consul/change-consul-port-availability.js

```diff
@@ -25,4 +25,12 @@ export async function changeConsulPortAvailability(enableNodePort) {
     logger.info(`consul service updated with result: [${JSON.stringify(updateResult)}]`)
 
     return updateResult
+}
+
+export async function getExternalPortState() {
```

Comment:
extract to an independent file, also rename method to `isConsulExternalPortEnabled` accordingly the file name should be `is-consul-external-port-enabled.js` and method should return true or false